core/libdivecomputer.c

```diff
@@ -992,14 +947,13 @@ static void verify_fingerprint(dc_device_t *device, device_data_t *devdata, cons
 {
```

Comment:
For some reason I can't comment on the comment above, but note that now
```
 * The fingerprint cache files contain the actual libdivecomputer
 * fingerprint, followed by 4 bytes of diveid data.
```
is not the complete story anymore.